### ISSUE TITLE
Control EVA Kerbals

### ISSUE THREAD

**Issue by djungelorm** [krpc/krpc#319]
Make the vessel class able to control them?

**Comment by mkluge** [comment]
Yes, please! My hardware console has already buttons for this ...

**Comment by djungelorm** [comment]
This could be useful reference: https://github.com/GER-Space/kOS-EVA
